include/macros.h

```diff
@@ -87,6 +87,11 @@
 
 extern GraphicsContext* oGfxCtx;
```

Comment:
trying to decide which to go with. i think getting rid of the o is probably a good idea now that we are going this route.
do we have an existing preference for single vs double underscore? I think i like how double looks but i dont care all too much.
what should the comment say?
```
// when appending to a display buffer use the DISP macros defined above to interface with the graphics context
```

Reply:
I guess its not just for appending. Think i need some help phrasing the explanation

Reply:
I think `__gfxCtx` should be fine. I don't have much opinion for the underscores but using double has one advantage because it will be easier to notice when reviewing code.

At least while we are actively decompiling, I think we should have a comment like
```
// __gfxCtx shouldn't be used directly, use DISP macros defined above when writing to display buffers
```

Reply:
sounds good, changes pushed